packages/@okta/vuepress-site/docs/reference/api/myaccount/index.md

```diff
@@ -739,6 +739,8 @@ If an invalid `method` is passed in the request body, the response returns a 400
 
 If the phone authenticator isn't enabled for `method` on the org, the response returns a 403 FORBIDDEN with error code E0000038.
 
+Disabling the `IDP_MY_ACCOUNT_API` feature flag in the org returns a 401 UNAUTHORIZED with error code E0000015.
```

Comment:
We need to avoid explicitly mentioning FFs in the docs :)
```suggestion
Disabling the `idp` version of the MyAccount API in the org returns a 401 UNAUTHORIZED with error code E0000015.
```